yugabyte/run-jepsen.py

```diff
@@ -249,7 +252,7 @@ def main():
     run_cmd(SORT_RESULTS_SH)
 
     start_time = time.time()
-    nemeses = NEMESES
+    nemeses = args.nemeses.split(' ')
```

Comment:
Actually, shouldn't this be `split(',')`?

Reply:
Ah, this is the key issue. When do we want to run nemesis one at a time, and when do we want to pass multiple down to the to the test all together. That is the real reason I had space and comma separators.  That is really too subtle. The way jenkins is using this, we should just pass these down, not loop through them.

Instead we only loop through  the workloads.